Gemfile

```diff
@@ -22,6 +22,8 @@ gem 'uglifier', '>= 1.3.0'
 
 gem 'devise'
 
+gem 'jquery-rails'
```

Comment:
Have you looked at removing the jquery script from layout?  Possible conflicts with the gem?